llvm/tools/llc/llc.cpp

```diff
@@ -410,6 +429,21 @@ int main(int argc, char **argv) {
 
   if (RemarksFile)
     RemarksFile->keep();
+  if (StringRef(OutputFilename).ends_with(".spv")) {
```

Comment:
This is a nit you can ignore, but ideally this would be done by a library call (which of course we can't do because spirv-val is out of tree), but maybe a comment explaining that would be useful for a future reader.

Reply:
Added a detailed comment for this. Thanks